### PR TITLE
#9Task. yulia.miasnikova

### PR DESCRIPTION
**Pull request by YuliaSirius**
https://travis-ci.org/github/YuliaSirius/js-assignments/builds/710687846

### CODE REVIEW

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -171,17 +184,16 @@ function partialUsingArguments(fn) {
...
 
-
 module.exports = {
-    getComposition: getComposition,
-    getPowerFunction: getPowerFunction,
-    getPolynom: getPolynom,
-    memoize: memoize,
-    retry: retry,
-    logger: logger,
-    partialUsingArguments: partialUsingArguments,
```
Comment (AlexMokin) [inline task/09-functions-n-closures-tasks.js]: keep original code formatting

File: task/09-functions-n-closures-tasks.js [hunk 2]
```diff
@@ -104,10 +101,19 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+  return function () {
+    while (attempts-- > 0) {
```
Comment (AlexMokin) [inline task/09-functions-n-closures-tasks.js]: You don't need 2 exit conditions. Leeav the one in the loop definition
image

**Review by AlexMokin — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by YuliaSirius** [comment]
https://travis-ci.org/github/YuliaSirius/js-assignments/builds/711066919

**Comment by AlexMokin** [comment]
accepted

**Pull request closed without merging**